### PR TITLE
en_torch OK

### PR DESCRIPTION
**Pull request by fullgrowngaming**
I ran the format script and it made the init vars look really weird. Not sure if that's what we wanted, but that's what happened when I ran the script, so I left it

### CODE REVIEW

**Comment by fig02** [comment]
can put a trailing comma on the last line to avoid that

File: src/overlays/actors/ovl_En_Torch/z_en_torch.c [hunk 1]
```diff
@@ -6,18 +6,26 @@
...
+    FLAGS,           
+    GAMEPLAY_KEEP,   
     sizeof(EnTorch),
-    (ActorFunc)EnTorch_Init,
-    (ActorFunc)NULL,
-    (ActorFunc)NULL,
+    (ActorFunc)EnTorch_Init, 
+    (ActorFunc)NULL,      
+    (ActorFunc)NULL, 
     (ActorFunc)NULL
```
Comment (Rozelette) [inline src/overlays/actors/ovl_En_Torch/z_en_torch.c:18]: Extra space at the end of lines

**Review by Rozelette — APPROVED** [review]
LGTM except for a minor formatting nitpick

**Review by Kenix3 — APPROVED** [review]
Looks good to me.
Very minor formatting nitpicks.
Build is OK

**Pull request merged**